torchci/pages/nightlies.tsx

```diff
@@ -0,0 +1,453 @@
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import dayjs from "dayjs";
+import ReactECharts from "echarts-for-react";
+import { EChartsOption } from "echarts";
+import {
+  GridRenderCellParams,
+  GridValueFormatterParams,
+} from "@mui/x-data-grid";
+import useSWR from "swr";
+import {
+  Grid,
+  Paper,
+  TextField,
+  Typography,
+  Stack,
+  Skeleton,
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  SelectChangeEvent,
+} from "@mui/material";
+import { LocalizationProvider, DateTimePicker } from "@mui/x-date-pickers";
+
+import { useEffect, useState } from "react";
+
+import { RocksetParam } from "lib/rockset";
+import { fetcher } from "lib/GeneralUtils";
+
+import TablePanel from "components/metrics/panels/TablePanel";
+import { durationDisplay } from "components/TimeUtils";
+
+function NightlyJobsRedPanel({ params, repo }: { params: RocksetParam[], repo: string }) {
```

Comment:
You could also consider using the existing `TimeSeriesPanel` component https://github.com/pytorch/test-infra/blob/main/torchci/components/metrics/panels/TimeSeriesPanel.tsx#L188 to simplify this whole function.  That function would use a line chart though, but I think line chart is a better choice for percentage data, thoughts?

Reply:
Tried it. Don't really like it. It looks more smooth. Bar chart in my opinion displaying better nightly run since each daily run. Line graph works better for something continuous. 
<img width="1531" alt="Screenshot 2023-06-21 at 12 50 15 PM" src="https://github.com/pytorch/test-infra/assets/7563158/26aeb59d-ba57-4c2a-8aef-7b89dc41c7c0">